po/ja.po

```diff
@@ -11269,151 +11331,162 @@ msgid ""
 "adb shell /data/local/tmp/hello_rust_with_dep\n"
 "```"
 msgstr ""
+"```shell\n"
+"m hello_rust_with_dep\n"
+"adb push \"$ANDROID_PRODUCT_OUT/system/bin/hello_rust_with_dep\" /data/local/"
+"tmp\n"
+"adb shell /data/local/tmp/hello_rust_with_dep\n"
+"```"
 
 #: src/android/aidl.md
 msgid ""
 "The [Android Interface Definition Language (AIDL)](https://developer.android."
 "com/guide/components/aidl) is supported in Rust:"
 msgstr ""
+"Rust では [Android インターフェース定義言語（AIDL）](https://developer."
+"android.com/guide/components/aidl) がサポートされています。"
 
 #: src/android/aidl.md
 msgid "Rust code can call existing AIDL servers,"
-msgstr ""
+msgstr "Rust コードは既存の AIDL サーバーを呼び出すことができます。"
 
 #: src/android/aidl.md
 msgid "You can create new AIDL servers in Rust."
-msgstr ""
+msgstr "Rust では新しい AIDL サーバーを作成できます。"
 
 #: src/android/aidl/interface.md
 msgid "AIDL Interfaces"
-msgstr ""
+msgstr "AIDL インターフェース"
 
 #: src/android/aidl/interface.md
 msgid "You declare the API of your service using an AIDL interface:"
-msgstr ""
+msgstr "サービスの API を宣言するには、AIDL インターフェースを使用します。"
 
 #: src/android/aidl/interface.md
 msgid ""
 "_birthday_service/aidl/com/example/birthdayservice/IBirthdayService.aidl_:"
 msgstr ""
+"_birthday_service/aidl/com/example/birthdayservice/IBirthdayService.aidl_:"
 
 #: src/android/aidl/interface.md src/android/aidl/changing.md
 msgid "/** Birthday service interface. */"
-msgstr ""
+msgstr "/** 誕生日サービスのインターフェース。*/"
 
 #: src/android/aidl/interface.md src/android/aidl/changing.md
 msgid "/** Generate a Happy Birthday message. */"
-msgstr ""
+msgstr "/** 「お誕生日おめでとう」というメッセージを生成します。*/"
 
 #: src/android/aidl/interface.md
 msgid "_birthday_service/aidl/Android.bp_:"
-msgstr ""
+msgstr "_birthday_service/aidl/Android.bp_:"
 
 #: src/android/aidl/interface.md
 msgid "\"com.example.birthdayservice\""
-msgstr ""
+msgstr "\"com.example.birthdayservice\""
 
 #: src/android/aidl/interface.md
 msgid "\"com/example/birthdayservice/*.aidl\""
-msgstr ""
+msgstr "\"com/example/birthdayservice/*.aidl\""
 
 #: src/android/aidl/interface.md
 msgid "// Rust is not enabled by default\n"
-msgstr ""
+msgstr "// Rust はデフォルトでは無効です。\n"
 
 #: src/android/aidl/interface.md
 msgid ""
 "Add `vendor_available: true` if your AIDL file is used by a binary in the "
 "vendor partition."
 msgstr ""
+"AIDL ファイルがベンダー パーティション内のバイナリで使用されている場合は、"
```

Comment:
Could it be that this block is changed in the main branch?

Reply:
I prefer merging this before then refreshing the translation and send additional PRs later.